api/tests/routes/native/v1/offers_test.py

```diff
@@ -278,6 +278,47 @@ def test_get_thing_offer(self, client):
         assert response.json["venue"]["isPermanent"]
         assert response.json["stocks"][0]["features"] == []
 
+    @pytest.mark.parametrize(
+        "provider_class,ff_name,ff_value,expected",
+        [
+            ("EMSStocks", "DISABLE_EMS_EXTERNAL_BOOKINGS", True, True),
+            ("EMSStocks", "DISABLE_EMS_EXTERNAL_BOOKINGS", False, False),
+            ("CGRStocks", "DISABLE_CGR_EXTERNAL_BOOKINGS", True, True),
+            ("CGRStocks", "DISABLE_CGR_EXTERNAL_BOOKINGS", False, False),
+            ("CDSStocks", "DISABLE_CDS_EXTERNAL_BOOKINGS", True, True),
+            ("CDSStocks", "DISABLE_CDS_EXTERNAL_BOOKINGS", False, False),
+            ("BoostStocks", "DISABLE_BOOST_EXTERNAL_BOOKINGS", True, True),
+            ("BoostStocks", "DISABLE_BOOST_EXTERNAL_BOOKINGS", False, False),
+            ("BoostStocks", "DISABLE_EMS_EXTERNAL_BOOKINGS", True, False),
+        ],
+    )
+    def test_get_sync_event_offer(self, client, provider_class, ff_name, ff_value, expected):
```

Comment:
```suggestion
    @pytest.mark.parametrize("provider_class", ["EMSStocks", "CGRStocks", "CDSStocks", "BoostStocks"])
    @override_features(DISABLE_EMS_EXTERNAL_BOOKINGS=True, DISABLE_CGR_EXTERNAL_BOOKINGS=True, DISABLE_CDS_EXTERNAL_BOOKINGS=True, DISABLE_BOOST_EXTERNAL_BOOKINGS=True)
    def test_get_sync_cinema_offer_with_disabled_providers(self, client, provider_class):
        ...


    @pytest.mark.parametrize("provider_class", ["EMSStocks", "CGRStocks", "CDSStocks", "BoostStocks"])
    @override_features(DISABLE_EMS_EXTERNAL_BOOKINGS=False, DISABLE_CGR_EXTERNAL_BOOKINGS=False, DISABLE_CDS_EXTERNAL_BOOKINGS=False, DISABLE_BOOST_EXTERNAL_BOOKINGS=False)
    def test_get_sync_cinema_offer_with_enabled_providers(self, client, provider_class):
        ...
```

plus il y a d'arguments dans le `parametrize`, plus le test devient difficile à lire et à maintenir. je suis aussi pas fan de passer `expected` dans les arguments de `parametrize`

le plus simple serait de séparer le test en 2, un pour le cas avec les synchros activées, l'autre pour celui où ils sont désactivés